### ISSUE TITLE
Point Geologic Unit Symbology 

### ISSUE THREAD

**Issue by egbunse** [StraboSpot/strabo-mobile#312]
Using the Geologic Units tag group, assign specific colors to each member the user inputs. When point spots are displayed on the main map, the font of their name is displayed in this color, so the user can get an idea of the spatial extent of rock units.

**Comment by jessicagood** [comment]
Now that we can assign colors to tags this becomes possible however I think it would be hard to read if we change the colors of label text on the map. I could change the color of the label text in the popups though. It won't make it as easy to see the see the spatial extent of rocks right away but if the map gets too cluttered and has too many colors it will be hard to see too. Any other thoughts on this?

**Comment by egbunse** [comment]
Is there a way of overwriting the pre-determined colors for points, lines, and polygons in map view? That might work, but I could see confusion when a feature has multiple tags... which color takes precedent?  I agree from looking at datasets again that if we changed label colors it would look very confusing. If you change the label text in the popups is there an easy way of enabling the selection/visibility of multiple popups at once? I know that might look cluttered on a phone screen but that would allow for some spatial extent to be seen. Maybe this is something for the online app only?

**Comment by btikoff** [comment]
HI-
  From my work with students, the rock type tab should take precedence on the field app.  On the desktop, it is more of an open question.
  Basil

**Comment by jessicagood** [comment]
Right now the only time colors in map view that are overwritten are if there is a geologic unit tag and a color assigned to that geologic unit tag. None of the other colors are overwritten.  Overwriting other colors may be a bigger issue to discuss.

I'll look into multiple popups or another way to more easily see the geologic unit extent.

**Comment by egbunse** [comment]
@jessicagood This weekend on the sedimentology trip, I was adding color to my geologic unit Tags and noticed a bug: when a point Spot is assigned a Tag with color, that point will no longer be visible on the main map (map won't load) or an image basemap (just the photo is visible). When the color is cleared from the Tag the Spot is again visible on maps.

**Comment by jessicagood** [comment]
@egbunse Good catch! Fixed with commit 358b5aa8ff4401854d7e057fa259d87d954b6b34 and the testing app has been updated.
